pspamm/scripts/max_arm.py

```diff
@@ -16,4 +16,4 @@ def getBlocksize(m , n, bk):
 
 
 def ARM_condition(bm, bn, bk):
-    return (bn+bk) * (bm / 2) + bn + 2 <= 32
+    return (bn+bk) * (bm / 2) + bn <= 32
```

Comment:
Should the `/ 2` here be a ceiling division as well?

Reply:
`bm` seems to be always a multiple of `2` as it is assigned the value of the loop variable `i` in the function `getBlocksize(m, n, bk)`, meaning that `bm / 2` will always be a "whole" float (i.e. `bm/2 = x.0`).
I can replace `bm / 2` with a ceiling division to keep it consistent with the `arm_sve` version.

The same applies to `max_hsw.py` and `max_knl.py`

Reply:
It would be good to have an integer division here IMO (regardless if ceil or floor)